lib/mix/tasks/init.ex

```diff
@@ -33,21 +34,6 @@ defmodule Mix.Tasks.Bootleg.Init do
 
   role :build, "build.example.com", workspace: "/tmp/bootleg/build"
 
-  # Phoenix has some extra build steps such as asset digesting that need to be done during
-  # compilation. To have bootleeg handle that for you, include the additional package
-  # `bootleg_phoenix` to your `deps` list. This will automatically perform the additional steps
-  # required for building phoenix releases.
-  #
-  #  ```
-  #  # mix.exs
-  #  def deps do
-  #    [{:distillery, "~> 1.5"},
-  #    {:bootleg, "~> 0.6"},
-  #    {:bootleg_phoenix, "~> 0.2"}]
-  #  end
-  #  ```
-  # For more about `bootleg_phoenix` see: https://github.com/labzero/bootleg_phoenix
-
```

Comment:
I think it's better to not include sample dependencies than to potentially include outdated versions by accident